foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py

```diff
@@ -92,7 +92,13 @@ def handle(self, *args, **options):
                     # GEOparse downloads files here and never cleans them up! Grrrr!
                     download_path = GEO_TEMP_DIR + experiment.accession_code + '_family.soft.gz'
                     # It's not a directory, but ignore_errors is useful.
-                    shutil.rmtree(download_path, ignore_errors=True)
+                    try:
+                        os.remove(download_path, ignore_errors=True)
+                    except:
```

Comment:
do we need the `try...except` if we have `ignore_errors=True`?

Also, `except...pass` makes me nervous.

Reply:
Oh good catch! `ignore_errors=True` isn't an option on os.remove(), I just forgot to delete it from the other method. I've changed this so we log so we can tell why we're failing to delete stuff.